curves/Cargo.toml

```diff
@@ -37,17 +37,18 @@ harness = false
 [dependencies.snarkvm-fields]
 path = "../fields"
 version = "0.9.13"
-default-features = false
 
 [dependencies.snarkvm-utilities]
 path = "../utilities"
 version = "0.9.13"
-default-features = false
 
 [dependencies.rand]
 version = "0.8"
 default-features = false
 
+[dependencies.rayon]
+version = "1"
+
```

Comment:
What does adding rayon here add to the crate?

Reply:
What's the reason `rayon` is added?